vdev/src/commands/release/push.rs

```diff
@@ -0,0 +1,34 @@
+use std::env;
+
+use anyhow::Result;
+use clap::Args;
+
+use crate::util;
+use crate::git;
+
+/// Pushes new versions produced by `make release` to the repository
+#[derive(Args, Debug)]
+#[command()]
+pub struct Cli {}
+
+impl Cli {
+    pub fn exec(self) -> Result<()> {
+        let version = env::var("VECTOR_VERSION").or_else(|_| util::read_version())?;
+        let version_minor = version.split('.')
+            .take(2)
+            .collect::<Vec<&str>>()
```

Comment:
If you add a `use itertools::Itertools;`, you can skip the `collect` here.

Reply:
I have implemented this locally and will push once I have resolved the other issues.